src/main/scala/devices/gpio/GPIOCtrlRegs.scala

```diff
@@ -21,4 +21,7 @@ object GPIOCtrlRegs {
   val out_xor     = 0x40
   val passthru_high_ie = 0x44
   val passthru_low_ie  = 0x48
+  val drive1      = 0x4c
```

Comment:
If it's enabled, please put `drive1` next to `drive0` in the memory map:
```
val drive0 = 0x14
val drive1 = 0x18
```

Reply:
Can we do something similar with `dsOffset` here that you did in the memory map?